src/Illuminate/Validation/Validator.php

```diff
@@ -1097,6 +1097,19 @@ protected function validateExists($attribute, $value, $parameters)
     }
 
     /**
+     * Validate the existence of an attribute value in a database table.
+     *
+     * @param  string  $attribute
+     * @param  mixed   $value
+     * @param  array   $parameters
```

Comment:
The phpdoc. You should have exactly 2 spaces either side of the type, and no more.